python/mqt/syrec/simulation_view/dialogs/simulation_run_json_import_dialog.py

```
@@ -0,0 +1,244 @@
+
+    from ..simulation_run_model import QtSimulationRunModel, SimulationRunModel
+
+from ...logger_utils import log_error_to_console, log_info_to_console
+from ...message_box_utils import MessageBoxType, show_and_request_ok_in_optionally_cancellable_notification
+from ..simulation_run_model import SimulationRunModel
+from ..workers.simulation_run_json_import_worker import SimulationRunJsonImportWorker
+from .base_progress_dialog import BaseProgressDialog
+
+
+class SimulationRunJsonImportDialog(BaseProgressDialog[SimulationRunJsonImportWorker]):
+    def __init__(self, parent: QtWidgets.QWidget):
```

Ruff (0.14.14)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)